app/frontend/src/api/models.ts

```diff
@@ -103,3 +104,21 @@ export interface SpeechConfig {
     isPlaying: boolean;
     setIsPlaying: (isPlaying: boolean) => void;
 }
+
+export type HistoryListApiResponse = {
+    items: {
+        id: string;
+        entra_id: string;
+        title?: string;
+        _ts: number;
```

Comment:
What/why is _ts? It's a funny variable name, so it could benefit from either an inline comment or a rename.

Reply:
`_ts` is a timestamp record that CosmosDB assigns by default.
We can define and assign a timestamp record defined in our application, but which is better?

https://learn.microsoft.com/en-us/rest/api/cosmos-db/documents

Reply:
Updated in 11c4d9f